test/queries_generator/aws_results/output10.csv

```diff
@@ -0,0 +1,10 @@
+1931-12-23T02:54:25.451925Z
```

Comment:
the Z thing cause a failure upon comparing results.
since results are static, and the Z thing has no meaning, it can be removed